ftml/src/parsing/rule/impls/block/mod.rs

```diff
@@ -61,8 +61,9 @@ pub struct BlockRule {
     /// `[[user aismallard]]` and `[[*user aismallard]]`.
     accepts_special: bool,
 
-    /// Whether this block wants its head and tail to be separated by newlines.
-    newline_separator: bool,
+    /// Whether this block allows its head and tail to be separated by newlines.
```

Comment:
While "allows" is probably clear enough, I'd prefer to specify explicitly that the newlines are optional (assuming this is the case)